lib/rules/no-duplicate-selectors/index.js

```diff
@@ -13,7 +13,7 @@ const resolvedNestedSelector = require("postcss-resolve-nested-selector")
 const ruleName = "no-duplicate-selectors"
 
 const messages = ruleMessages(ruleName, {
-  rejected: selector => `Unexpected duplicate selector "${selector}"`,
+  rejected: (selector, firstDuplicateLine) => `Unexpected duplicate selector "${selector}" is also at line ${firstDuplicateLine}`,
```

Comment:
It's very confusing. User might understand, that duplicated selector at line `firstDuplicateLine`, rather at line stylelint reports for this message.

```
16:1 Unexpected duplicate of selector "a" at line 3
```

I would think duplicate selector is at line 3. And it'd take some time to understand that it's at line 16.

What about `Unexpected duplicate selector "${selector}", first occurrence at line ${firstDuplicateLine}`? Maybe “occurrence” could be replaced with other word.